features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,38 @@
+Given(/^I'm on the "([^"]*)" page$/) do |page|
+  case page
+    when 'home'
+      visit '/'
+    when 'Sign up'
+      visit 'users/sign_up'
+  end
+end
+
+And(/^I click on "([^"]*)"$/) do |unicorn|
+  click_link_or_button unicorn
+end
+
+Then(/^I should be on the "([^"]*)" page$/) do |page|
+  case page
+    when 'Sign up'
+      expect(current_path).to eq '/users/sign_up'
+    end
+end
+
+And(/^I fill in "([^"]*)" with "([^"]*)"$/) do |form, content|
+  fill_in(form, with: content)
```

Comment:
The variable name is a bit off... It's a field not a form. A minor details but it's rather important.